.github/workflows/drizzle_ci.yml

```diff
@@ -0,0 +1,81 @@
+name: Drizzle Package Continuous Integration
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+jobs:
+  build:
+    name: ${{ matrix.name }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - name: Python 3.9 Test
+            pyver: 3.9
+            dependencies: 
+
+          - name: Old Dependencies
+            pyver: 3.6.10
+            dependencies: |
+              python -m pip install numpy~=1.16.0
+
+          - name: Development Build
+            pyver: 3.8.5
```

Comment:
This dev build should probably run on Python 3.9 too.

Reply:
I updated to Python 3.9.